troposphere/static/js/components/modals/nullProject/NullProjectMigrateResourceModal.react.js

```diff
@@ -169,7 +168,12 @@ define(
       },
 
       renderBody: function(){
-        if(this.state.projects) {
+        if(this.state.projects == null) {
```

Comment:
Believe style-wise if(this.state.projects) a bit cleaner but nothing wrong with being explicit.